packages/adapter-neon/src/neon.ts

```diff
@@ -121,13 +122,23 @@ class NeonTransaction extends NeonWsQueryable<neon.PoolClient> implements Transa
   }
 }
 
+export type PrismaNeonOptions = {
+  schemaName?: string
```

Comment:
I might suggest that we fully rename this from `schemaName` to `schema` in the public API to be consistent with the `schema` connection string parameter that we have in our connection strings.

No necessary to adapt yet, ad a higher level discussion is going on and I think this would be a simple search/replace.

Reply:
It was a quick change, done.